### ISSUE TITLE
gulp watch not triggering (windows 8.1)

### ISSUE THREAD

**Issue by mathiasvalentin** [vigetlabs/blendid#211]
I run 'npm run gulp' and everything looks good, but making changes to script, style, images and html doesn't trigger the gulp watch. 
Any ideas on how to debug or fix this problem?

**Comment by greypants** [comment]
Hm... it's possible it's a Windows issue with gulp-watch... see if anything here looks relevant: https://github.com/floatdrop/gulp-watch/issues

I'll verify that I didn't break watching up in a recent commit.

**Comment by greypants** [comment]
Also, I'm admittedly using gulp-watch a little weird.

**Comment by mathiasvalentin** [comment]
@greypants Thanks, for the fast reply! 
Changing from **watch** to **gulp.watch** worked for html, css etc. 
https://github.com/floatdrop/gulp-watch/issues/192

But still having the same problem with .js and webpack.watch. Ideas are welcome :)

**Comment by adambuczynski** [comment]
On windows, patterns with `**/*.js` don't seem to trigger any watch changes.

**Comment by chaelli** [comment]
not sure if this is the case here - but we had the issue that watch didn't trigger because the path was too long - moving the project into the drive-root solved the issue

**Comment by greypants** [comment]
Hm. Interesting...

**Comment by highergroundstudio** [comment]
Had the same problem on as @mathiasvalentin and changing to gulp.watch seems to be working now.

Running on windows 10

**Comment by furkanpoyraz** [comment]
Having the same issue as you guys on Ubuntu & Windows 10.

**Comment by fyllepo** [comment]
I run this perfectly on OSX, but at home I had the same bug as @mathiasvalentin on windows 8.1 Pro.

Changing to gulp.watch however didn't rectify the problem.

I've fixed mine however through updating and relocating my project folder, in order:

1. Updated Node to latest stable release.
2. Installed Python 2.7.6 at C:/ and set up python environment variable (namely because I had errors relating to python in previous `npm install` attempt).
3. Installed VS Code [https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs#DownloadFamilies_2] (had similar issues with node-gyp depending on VCBuild.exe).
4. Recloned my repository to start fresh, this time to root drive C:/ to help mitigate path length issues that frequently occur with Node on windows.
5. Ran `npm install` again, and now all gulp-watch is working perfectly.... much to my surprise.

Hope that helps someone, or gives you a hand debugging.
